src/dashboard/index.html

```diff
@@ -19,8 +19,10 @@ <h3>Bridge</h3>
         <div>
             <i>Start Bridge with emulator support.</i><br>
             <button onclick="bridgeStart('2.0.31')">Start 2.0.31</button>
-            <button onclick="bridgeStop()">Stop</button>
+            <button onclick="bridgeStart('2.0.31-arm64')">Start 2.0.31-arm64</button>
```

Comment:
With rising number of bridges we could consider generating the list of available bridges from the python code (same as we are generating selection of available emulators)

Reply:
Good poin, i will try to change it

Reply:
@grdddj do you think we could ask you for this change? We would merge as is now (after some changes Tali just posted) and you would do this small change in a different PR then? We'll make you an issue :)

Reply:
Sure, it sounds like an interesting task :)

Reply:
Great moving to https://github.com/trezor/trezor-user-env/issues/147.